workers/unity/Packages/com.improbable.gdk.transformsynchronization/Components/BufferedTransform.cs

```diff
@@ -7,6 +8,7 @@ public class BufferedTransform : Component
     {
         public System.Collections.Generic.List<SpatialOSTransform> TransformUpdates;
         public SpatialOSTransform LastTransformSnapshot;
+        public BlittableBool IsInitialised;
```

Comment:
Is this actually bridge related? Is it sending us partial updates or something?

Reply:
`bridge_v2` has diffing turned on by default (as of today)